### PR TITLE
1507-Revise-Theme-Selector-Controls

### PR DESCRIPTION
**Pull request by giduac**
1507-Revise-Theme-Selector-Controls
[issue 1507](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1507)
Fourth part:
- Remove all references to the ReportSelectedThemeIndex property throughout alpha.
- Update the change log for this ticket.

compile-results

### CODE REVIEW

**Comment by Smurf-IV** [comment]
@giduac conflicts?

**Comment by giduac** [comment]
> @giduac conflicts?

Looked cool when I posted it yesterday....

File: Documents/Changelog/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Implemented [#1507](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1507), **[Breaking Change]** `KryptonThemeComboBox`, `KryptonThemeListBox` & `KryptonRibbonGroupThemeComboBox`:
```
Comment (Smurf-IV) [inline Documents/Changelog/Changelog.md:6]: Breaking change description goes into a different section

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

**Comment by giduac** [comment]
@Smurf-IV 
And again please !!

File: Documents/Changelog/Changelog.md (hunk 1 above)
Comment (giduac) [reply Documents/Changelog/Changelog.md:6]: > Breaking change description goes into a different section

What about the others in the list. That's how I came to putting it in with the rest.

Some help please as I don't know where that would be.

**Pull request merged**